src/workshop-comments.njk

```diff
@@ -0,0 +1,54 @@
+---
+pagination:
+    data: collections.workshops
+    size: 1
+    alias: workshopItem
+permalink: "initiatives/{{ workshopItem.id }}/"
+---
+{% extends 'layouts/base.njk' %}
+
+{% set pageTitle = workshopItem.title %}
+
+{% block content %}
+	<article class="workshop-page">
+		<h1>{{ workshopItem.title | safe }}</h1>
+		{% if workshopItem.imageUrl %}
+			<img class="workshop-image" src="{{ workshopItem.imageUrl | safe }}" alt="{{ workshopItem.imageAlt }}">
+		{% endif %}
+
+		<div class="description">{{ workshopItem.fullDescription | safe }}</div>
+
+		<form id="comment-form">
+			<h3>Comments</h3>
+			<div class="form-control">
+				<span>*</span><p class="asterisk-def"> Indicates required field</p>
+			</div>
+
+			<div class="form-control">
+				<span>*</span><label for="name"><h4>Name</h4></label><span class="required">Required</span>
+				<input id="name" type="text" name="name" :value="nameTerm" placeholder="" aria-label="Enter your name">
+			</div>
+
+			<div class="form-control">
+				<span>*</span><label for="comment"><h4>Post a comment or question</h4></label><span class="required">Required</span>
+				<textarea id="comment" type="text" name="comment" :value="commentTerm" placeholder="" aria-label="Enter your comment"></textarea>
+				<input type="hidden" name="workshopId" id="workshopId" value="{{ workshopItem.id }}">
+				<button type="submit">Post a comment</button>
+			</div>
+		</form>
+
+		<div class="submitted-comment-message">
```

Comment:
Status areas announcing successful or failed comment submissions should be marked as [ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) so that ATs could notify users what happens.

Please test with screen readers such as VoiceOver on Safari to make sure the submission result is announced after clicking "post a comment" button. Thanks.